code/modules/holodeck/computer.dm

```diff
@@ -396,6 +396,8 @@ GLOBAL_LIST_INIT(typecache_holodeck_linked_floorcheck_ok, typecacheof(list(/turf
 		return
 	for(var/obj/item/to_be_nerfed in spawned)
 		to_be_nerfed.damtype = nerf_this ? STAMINA : initial(to_be_nerfed.damtype)
+		to_be_nerfed.force = nerf_this ? 0 : initial(to_be_nerfed.force)
```

Comment:
why are you changing the force value? theyre already changed to only do stamina damage

Reply:
Reverted with https://github.com/tgstation/tgstation/pull/64200/commits/47c8a5c6070f359773de1febdb773cf36f7f5c44

I think I mixed up the `force` and `throwforce` vars since neither were documented.